crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -583,6 +584,7 @@ mod tests {
         assert_eq!(*world.get::<Parent>(child1).unwrap(), Parent(parent));
         assert_eq!(*world.get::<Parent>(child2).unwrap(), Parent(parent));
 
+        // Why are those assertions repeated?
```

Comment:
Probably a mismerge. Can you please delete instead of adding a comment?

Reply:
I wasn't willing to remove those asserts before someone reviewed the PR and told me they were not here for a reason I was overlooking ([this is called "Chesterton's Fence"](https://en.wikipedia.org/wiki/Wikipedia:Chesterton's_fence))

Reply:
Yes absolutely, not blaming you here, that was the right call. But I think we both agree that this is useless and unlikely to have any real use? So I was just ACK'ing on the removal. Or do you want to wait for a second reviewer's opinion too? Im fine with that.

Reply:
We should remove the duplicate assertions.